pyuntl/compare.py

```diff
@@ -1,15 +1,16 @@
-def compare_elements(hash_dict1, hash_dict2):
-    """Check if any elements have been added, removed or modified."""
+def compare_elements(prev_hash_dict, current_hash_dict):
+    """Compare elements that have changed between hash_dict1 and hash_dict2.
```

Comment:
So since you changed the var names, this needs to be updated to match :smile:

Reply:
ugh...my bad. I first thought is it necessary and then though no harm in adding it and copied and pasted it. I'll fix it. Thanks!

Reply:
Also, the formatting should use the [multi-line docstring convention](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).